BGAnimations/ScreenEvaluation common/PerPlayer/ItlFile.lua

```diff
@@ -0,0 +1,162 @@
+local player = ...
+local pn = ToEnumShortString(player)
+
+local year = Year()
+local month = MonthOfYear()+1
```

Comment:
Gotta love that `MonthOfYear()` is 0-based when `DayOfMonth()` is 1-based. :rofl:

Reply:
#juststepmaniathings